data-protocols/ids/ids-api-transfer/src/main/java/org/eclipse/dataspaceconnector/ids/api/transfer/ArtifactRequestController.java

```diff
@@ -112,17 +113,24 @@ public Response request(ArtifactRequestMessage message) {
 
 
         // TODO this needs to be deserialized from the artifact request message
-        var destinationMap = (Map<String, Object>) message.getProperties().get(ArtifactRequestController.DESTINATION_KEY);
+        Map<String, Object> messageProperties = message.getProperties();
+        var destinationMap = (Map<String, Object>) messageProperties.get(ArtifactRequestController.DESTINATION_KEY);
         var type = (String) destinationMap.get("type");
 
         Map<String, String> properties = (Map<String, String>) destinationMap.get("properties");
```

Comment:
Looking at this it seems we have DataDestination properties and DataRequest properties. Consider renaming variables accordingly.

Reply:
without wanting a huge discussion on that, my vote goes for `requestProperties` and `destinationProperties`. I would only do the renaming for the new added properties though.

Reply:
_requestProperties_ can be misleading because the properties are from ArtifactRequestMessage and they are not directly transferred to DataRequest (only the properties inside _messageProperties_ are). So I'd leave _messageProperties_ here. 
But agree, change from _properties_ to _destinationProperties_ can improve the readability.

Reply:
Renamed properies to destinationProperties